internal/pkg/deploy/cloudformation/stack/testdata/workloads/backend/http-autoscaling-params.json

```diff
@@ -11,6 +11,7 @@
 		"LogRetention": "30",
 		"ContainerPort": "8080",
 		"EnvFileARN": "",
+		"ArtifactKeyARN": "",
```

Comment:
Can we in at least one integ test verifying `ArtifactKeyARN` can be rendered correctly with the app stackset resource info?